app/models/steps/excise.rb

```diff
@@ -0,0 +1,74 @@
+module Steps
+  class Excise < Steps::Base
+    STEPS_TO_REMOVE_FROM_SESSION = %w[].freeze
+
+    include CommodityHelper
+
+    attribute :measure_type_id, :string
+    attribute :additional_code, :string
+
+    validates :measure_type_id, presence: true
+    validates :additional_code, presence: true
```

Comment:
Does there need to be additional validation around the codes supplied by the user?

Reply:
Talked this through. :+1: